### ISSUE TITLE
Studio Live III - Presonus Audio Mixer

### ISSUE THREAD

**Issue by xinbatad** [bitfocus/companion-module-requests#1653]
Can we have these please. I know someone was working on it but it is good if its release in beta or official release. Thanks!

**Comment by imediawebStudio** [comment]
I agree with you it would be great to be able to control the Presonus family (StudioLive III) with companion

**Comment by Teckiee** [comment]
I'd like this too. There doesn't seem to be an official api, but Mixing Station got it in their app somehow.

**Comment by primepeter** [comment]
https://github.com/featherbear/bitfocus-companion-module-presonus-studiolive/tree/companion_v3

**Comment by leonreucher** [comment]
@primepeter please visit the Bitfocus Slack https://bfoc.us/54brzjjkk9 and get the module added to Companion.

Duplicate of #52
